schema.ts

```diff
@@ -568,6 +568,40 @@ export interface StringDateTimeSchema extends BaseStringSchema<ValueHintType.Dat
   timeFormat?: string;
 }
 
+/**
+ * State of outline on images that can be used with a {@link ImageSchema}.
+ */
+export enum Outline {
+  /** Image outline is disabled. */
+  Disabled = 'disabled',
+  /** Image outline is applied. */
+  Default = 'default',
+}
+
+/**
+ * State of rounded corners on images that can be used with a {@link ImageSchema}.
+ */
+export enum RoundedCorners {
+  /** Image rounded corners are disabled. */
+  Disabled = 'disabled',
+  /** Image rounded corners are applied. */
+  Default = 'default',
+}
+
+/**
+ * A schema representing a return value or object property that is provided as a string,
+ * which Coda should interpret as an image.
+ */
+ export interface ImageSchema extends BaseStringSchema<
```

Comment:
Now that you have an explicit schema for these hint types, I think you'll want to remove them from here: https://github.com/coda/packs-sdk/blob/main/schema.ts#L623

Reply:
the imageProperty in the object schema requires that the property contains one of those hints (https://github.com/coda/packs-sdk/blob/main/schema.ts#L878) so a lot of errors were brought up around that. Do you have any suggests for how can I can resolve that?

Reply:
Oh, yeah it's because you have to implement a new validator for your schema. This should do it, you can copy this into this PR:

https://github.com/coda/packs-sdk/commit/86c3dcfdcd56b02ee8f6d2218fffafb15822d564#diff-8f115faeb93eef8da8c710afe7051c558275f28ab6015e1a6bd21f2f7a599f51

Reply:
Mind including these tests too, to make sure the new fields validate? https://github.com/coda/packs-sdk/commit/fc01a240e5a0f3c1a6ff94ba073cc4826801e776#diff-e51cad99b49dae3d3ff0b09db2a20be1dfbef9e67950e077b326088856860551